config/api.yml

```diff
@@ -105,38 +105,6 @@
       :get:
       - :name: read
         :identifier: alert_status_show
-  :arbitration_profiles:
-    :description: Arbitration Profiles
-    :identifier: ems_cloud_admin
-    :options:
-    - :collection
-    :verbs: *gpppd
-    :klass: ArbitrationProfile
-    :collection_actions:
-      :get:
-      - :name: read
-        :identifier: arbitration_profile_show_list
-      :post:
-      - :name: query
-        :identifier: arbitration_profile_show_list
-      - :name: create
-        :identifier: arbitration_profile_new
-      - :name: edit
-        :identifier: arbitration_profile_edit
-      - :name: delete
-        :identifier: arbitration_profile_delete
-    :resource_actions:
-      :get:
-      - :name: read
-        :identifier: arbitration_profile_show
-      :post:
-      - :name: edit
-        :identifier: arbitration_profile_edit
-      - :name: delete
-        :identifier: arbitration_profile_delete
-      :delete:
-      - :name: delete
-        :identifier: arbitration_profile_delete
```

Comment:
I wouldn't remove from API unless it causes a problem.

This is part of E-release http://manageiq.org/docs/reference/latest/api/reference/arbitration_profiles so yes it would require an incompatible version bump. Also, this is partial removal as there's also code specific to this collection.

Probably another reason to have API centric roles and not share the UI ones.